satisfaction_analytics_zendesk_support/dbt_project/models/sources/zendesk_support_sources.yml

```diff
@@ -0,0 +1,316 @@
+version: 2
+
+sources:
+  - name: zendesk_support
+    # Use your BigQuery project ID
+    database: collab-test-bishal
```

Comment:
@bishalbera could you use a more generic name for the database? such as `my_project_id`?